tasks/build/index.js

```diff
@@ -1,35 +1,52 @@
 "use strict"
 
 const { cli } = require(`@nodeguy/cli`)
-const options = require(`./options.json`)
+const specification = require(`./options.json`)
 const path = require(`path`)
+const R = require(`ramda`)
 const shell = require(`shelljs`)
 const untildify = require(`untildify`)
 
-cli(options, async ({ commit, gaia, platform, "skip-pack": skipPack }) => {
+cli(specification, options => {
+  const { commit, gaia, network } = options
+
   shell.exec(`docker build --tag cosmos/voyager-builder .`, {
     cwd: __dirname
   })
 
-  const builds = path.resolve(__dirname, "../../builds")
-  shell.mkdir(`-p`, builds)
+  const resolved = R.map(R.pipe(untildify, path.resolve), {
```

Comment:
Do we need a plugin just for these lines?

Reply:
[Ramda](http://ramdajs.com/) is the best and most popular functional programming library for JavaScript.  It will find its way into every one of our projects and should probably just be included automatically by default.

Reply:
I usually try to avoid libraries because they are almost always an overhead to understand the code. 
Either way I think the line could benefit from some comments. I.e. what does untildify do?

Reply:
I also see the use of libraries as a cost but the value of Ramda is well worth it.  Many of its methods (such as `map` and `pipe` above) are the same in all functional programming languages and so are universally understandable.

I added more comments to the whole file.

Reply:
Note: We also talked about already having Lodash as a library in place. I guess there it would be `_.mapValues({ ... }, _.compose(untildify, path.resolve)`. Would this also be feasible?

Reply:
More important on the other hand side is merging this. I just want to reduce the time spent on cleaning up code when we decide on one functional library.

Reply:
Ramda is superior to Lodash.  See [The Philosophy of Ramda](http://fr.umio.us/the-philosophy-of-ramda/#whynotjustuse) (or for an in-depth explanation [Hey Underscore, You're Doing It Wrong!](https://www.youtube.com/watch?v=m3svKOdZijA)).  We should switch from Lodash to Ramda.

Reply:
I don't want to waste too time debating which lib is better--let's be practical about this. Lodash is imported a grand total of six times into Voyager, so it's easy to replace with Ramda if we want to use the latter instead. I've never used Ramda before but the docs make it look simple enough. I'm fine with switching.

Next time we want to replace a library in Voyager, we need to make a separate issue for the the lib switch instead of trying to do two things in one PR. This slows down reviews and merges. We can put this in the coding guide.

Reply:
I understand and agree with your concern but I'm not trying to replace a library in this PR.  I wasn't even aware that we were using Lodash before I wrote this.

Do you want me to rewrite the PR using Lodash instead of Ramda?  To me that feels like the slower path.

Reply:
I rewrote the PR using Lodash.